docs/qml/ml_tools.md

```diff
@@ -111,14 +116,16 @@ from qadence import hamiltonian_factory, hea, feature_map, chain
 from qadence.models import QNN
 from qadence.ml_tools import  TrainConfig, train_with_grad, to_dataloader
 
-n_qubits = 2
+DEVICE = torch.device('cuda') if torch.cuda.is_available() else torch.device('cpu')
```

Comment:
Would it make sense to showcase runtime or precisions comparisons with running the highest precision ?

Reply:
i was thinking the same but i think its not the right place here, this .md file is for introducing the user to ml_tools and adding benchmarks would just overload them

Reply:
OK maybe an example notebook ?

Reply:
added an example